beacon-chain/rpc/eth/light-client/structs.go

```diff
@@ -0,0 +1,35 @@
+package lightclient
+
+import (
+	"github.com/prysmaticlabs/prysm/v4/beacon-chain/rpc/apimiddleware"
+)
+
+type LightClientBootstrapResponse struct {
+	Version string                `json:"version"`
+	Data    *LightClientBootstrap `json:"data"`
+}
+
+type LightClientBootstrap struct {
+	Header                     *apimiddleware.BeaconBlockHeaderJson `json:"header"`
+	CurrentSyncCommittee       *apimiddleware.SyncCommitteeJson     `json:"current_sync_committee"`
```

Comment:
these two fields should be of the newly created types that I mentioned in other comments, not the ones in `apimiddleware`

Reply:
Patched.